packages/create-block-interactive-template/README.md

```diff
@@ -10,7 +10,7 @@ This block template can be used by running the following command:
 npx @wordpress/create-block --template @wordpress/create-block-interactive-template
 ```
 
-It requires Gutenberg 17.5 or higher.
+It requires at least WordPress 16.5 or Gutenberg 17.7.
```

Comment:
I think something is going wrong here `WordPress 16.5` Maybe it's 6.5

Reply:
Whoops, thanks for spotting that.
```suggestion
It requires at least WordPress 6.5 or Gutenberg 17.7.
```